bin/postgres_common/sqlrunner/start.sh

```diff
@@ -13,9 +13,13 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 
-if [[ -v PGAUDIT_ANALYZE ]]
-then
```

Comment:
Is the original `pgaudit` code still available for both the postgres and postgres-ha containers?

Reply:
Yes. The two different files have been compacted into one script: https://github.com/CrunchyData/crunchy-containers/blob/f7236965d8d022306ebddd9b8c15e42bf1aabdd6/bin/postgres_common/modules/pgaudit.sh

The postgres and postgres-ha versions used different env vars for the `pgdata/pg_log` directory. This has been updated to this: `${PATRONI_POSTGRESQL_DATA_DIR:-$PGDATA}/pg_log`